build/codeformat/formatimports.go

```diff
@@ -158,7 +163,7 @@ func formatGoImports(contentBytes []byte) ([]byte, error) {
 }
 
 // FormatGoImports format the imports by our rules (see unit tests)
-func FormatGoImports(file string) error {
+func FormatGoImports(file string, changedFiles, writeFile, outputDiff bool) error {
```

Comment:
```suggestion
func FormatGoImports(file string, changedFiles bool, writeFile bool, outputDiff bool) error {
```

Reply:
This is explicitly disallowed by gofumpt formatting style.

Reply:
Huh, seems like an anti-pattern to me. What is the reasoning behind it? Do types inherit from the right side?